### PR TITLE
adds primary 2 source

### PR DESCRIPTION
**Pull request by nate-moo**
primary 1 source will be worked on later

### CODE REVIEW

**Comment by AbdullahM0hamed** [comment]
435: Unable to authenticate PR at present

File: anime_downloader/sites/animixplay.py [hunk 1]
```diff
@@ -25,18 +26,43 @@ def search(cls, query):
     def _scrape_episodes(self):
```
Comment (Blatzar) [inline anime_downloader/sites/animixplay.py:26]: Most of this should really be in AniMixPlayEpisode, this is only to get the episode links

File: anime_downloader/sites/animixplay.py [hunk 2]
```diff
@@ -25,18 +26,43 @@ def search(cls, query):
...
+                data={data_id:post_id}).json())
+            logger.debug(data)
+            if '/v4/' in self.url:
+                return [x for x in data]
+            elif '/v2/' in self.url:
+                for x in data: 
+                    logger.debug("")
+                    raise NotImplementedError
+            else:
+                logger.error("What have you done? you managed to bypass the first check. good job and have a pat on the back. BTW this wasnt supposed to happen")
```
Comment (Blatzar) [inline anime_downloader/sites/animixplay.py:49]: lmao

File: anime_downloader/sites/animixplay.py [hunk 3]
```diff
@@ -25,18 +26,43 @@ def search(cls, query):
...
+            url_id = str.encode(self.url.split("/")[4])
+            post_id = f'NaN{base64.b64encode(url_id).decode()}N4CP9Eb6laO9N'.encode()
+            post_id = base64.b64encode(post_id).decode()
+            data_id = 'id2' if '/v4/' in self.url else 'id'
+            data = (helpers.post('https://animixplay.com/raw/2ENCwGVubdvzrQ2eu4hBH',
+                data={data_id:post_id}).json())
+            logger.debug(data)
+            if '/v4/' in self.url:
+                return [x for x in data]
+            elif '/v2/' in self.url:
```
Comment (Blatzar) [inline anime_downloader/sites/animixplay.py:44]: Implementing v2 is a not a challenge, will do it myself when I'm done/bored with MAL implementation

File: anime_downloader/sites/animixplay.py (hunk 1 above)
Comment (Blatzar) [reply anime_downloader/sites/animixplay.py:26]: Upon closer inspection there's no better way to do this

File: anime_downloader/sites/animixplay.py (hunk 3 above)
Comment (nate-moo) [reply anime_downloader/sites/animixplay.py:55]: ok

**Pull request merged**